src/Visualization/Python/PlotAlongLine.py

```diff
@@ -0,0 +1,249 @@
+#!/usr/bin/env python
+
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+import logging
+import os
+
+import click
+import matplotlib.pyplot as plt
+import numpy as np
+import rich
+
+import spectre.IO.H5 as spectre_h5
+from spectre.IO.Exporter import interpolate_to_points
+
+logger = logging.getLogger(__name__)
+
+
+def _parse_point(ctx, param, value):
+    if not value:
+        return None
+    return np.array(list(map(float, value.split(","))))
+
+
+@click.command(name="plot-along-line")
+@click.argument(
+    "h5_files",
+    nargs=-1,
+    type=click.Path(exists=True, file_okay=True, dir_okay=False, readable=True),
+)
+@click.option(
+    "--subfile-name",
+    "-d",
+    help="Name of subfile within h5 file containing volume data to plot.",
+)
+@click.option(
+    "--list-vars",
+    "-l",
+    is_flag=True,
+    help="Print available variables and exit.",
+)
+@click.option(
+    "--var",
+    "-y",
+    "vars",
+    multiple=True,
+    help=(
+        "Variables to plot. List any tensor components "
+        "in the volume data file, such as 'Shift_x'."
+    ),
+)
+@click.option(
+    "--step",
+    type=int,
```

Comment:
You have code in `Clip.py` to allow `first` and `last` as values too. Maybe use that to be consistent?